evap/evaluation/tests/test_models.py

```diff
@@ -670,6 +670,18 @@ def test_get_sorted_due_evaluations(self):
         sorted_evaluations = student.get_sorted_due_evaluations()
         self.assertEqual(sorted_evaluations, [(evaluations[1], 0), (evaluations[0], 0), (evaluations[2], 1)])
 
+    def test_correct_sorting(self):
+        baker.make(
+            UserProfile,
+            last_name=iter(["B", "A", None]),
```

Comment:
Strictly, this doesn't test that a) we `Lower` the fields and b) that there are more sorting keys than just the last name